src/Tizen.NUI/src/public/Rendering/VertexBuffer.cs

```diff
@@ -54,7 +54,7 @@ internal VertexBuffer(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr,
 
         public void SetData<VertexType>(VertexType[] vertices) where VertexType : struct
         {
-            if (null == vertices)
+            if (null == vertices || vertices.Length == 0)
```

Comment:
Minor. Could you please add comments about exception (line 52) that we can throw exception if certices length is zero?

Reply:
Done.
Thanks.